### PR TITLE
Create Book model

### PR DESCRIPTION
**Pull request by martinremis**
## Summary
* Create Book model

## Trello card
https://trello.com/c/5Ua8gIi3/14-crear-modelo-de-book

### CODE REVIEW

File: src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
+package wolox.training.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Column;
+
+@Entity
+public class Book {
+
+  @Column()
```
Comment (rodrigofranc) [inline src/main/java/wolox/training/models/Book.java]: The `()` are not necessary

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
+package wolox.training.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Column;
+
+@Entity
+public class Book {
```
Comment (rodrigofranc) [inline src/main/java/wolox/training/models/Book.java:13]: Add id to the entity, for unike. As unique key

Comment (rodrigofranc) [inline src/main/java/wolox/training/models/Book.java:13]: Add some java docs, is not part of the card but they help a lot

**Review by rodrigofranc — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/models/Book.java (hunk 2 above)
Comment (martinremis) [reply src/main/java/wolox/training/models/Book.java:13]: I'm not sure how to do it, is this ok? I've used a plugin to generate it.

Comment (rodrigofranc) [reply src/main/java/wolox/training/models/Book.java:13]: yes, is not a bad idea use a plugin for java docs. Maybe when you have some method with a lot of business logic you should write it yourself

**Review by rodrigofranc — DISMISSED** [review]

**Review by Mujics — DISMISSED** [review]

**Review by Mujics — APPROVED** [review]

**Review by rodrigofranc — APPROVED** [review]

**Pull request merged**